rotkehlchen/db/drivers/gevent.py

```diff
@@ -15,6 +15,7 @@
 import gevent
 from pysqlcipher3 import dbapi2 as sqlcipher
 
+from rotkehlchen.db import settings
```

Comment:
Though I asked you to remove this, never import stuff like this. You are importing the entire module here. You only needed CachedSettings.